checker/tests/nullness-safedefaultsbytecode/BasicTest.java

```diff
@@ -0,0 +1,27 @@
+import org.checkerframework.framework.qual.AnnotatedFor;
+
+// Tests that the -AsafeDefaultsForUnannotatedBytecode option does not
+// affect defaulting nor suppress errors in source code.
+
+class BasicTest {
+  void f() {
+    g("");
+  }
+
+  void g(String s) {}
+}
+
+// @AnnotatedFor("nullness")
```

Comment:
This should be a top-level annotation, not in comments. Otherwise the test will not work with the standard OpenJDK javac.